backend/src/mirrors_qa_backend/extract.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any
+from urllib.parse import urlsplit
+
+import pycountry
+import requests
+from bs4 import BeautifulSoup, NavigableString
+from bs4.element import Tag
+
+from mirrors_qa_backend import logger, schemas
+from mirrors_qa_backend.exceptions import MirrorsExtractError
+from mirrors_qa_backend.settings import Settings
+
+
+def get_current_mirrors() -> list[schemas.Mirror]:
+    """
+    Returns list of current mirrors from the mrirors url.
+
+    Raises MirrorsExtractError if the parser is unable to extract the mirrors
```

Comment:
Appropriate formatting (for doc-generating tools based on doctring):

```
"""  Current mirrors from the mirrors url.

    Raises:
            MirrorsExtractError: parser unable to extract the mirrors from page
                Page DOM has been updated and parser needs an update as well
```

Please note that this is misleading as we are also raising a bunch of requests exception should there be any network issue or if status is not 200